web/components/contract/like-button.tsx

```diff
@@ -170,7 +180,12 @@ export const LikeButton = memo(function LikeButton(props: {
           short={true}
         />
       )}
-    </>
+      {showTotalLikesUnder && (
+        <div className="mx-auto -mt-1.5 h-6 text-sm text-white drop-shadow-sm disabled:opacity-50">
+          {totalLikes > 0 ? totalLikes : ''}
```

Comment:
Might include a label "likes"? I believe TikTok uses labels, but I'm not sure about this case...

Reply:
Tiktok labels it if there's no likes yet, I kind of don't like the way it looks and I think it's self explanatory enough?